flow-typed/debugger-html.js

```diff
@@ -1,309 +0,0 @@
-// @flow
```

Comment:
`eslint-import` did not like this pattern. I tried several things, but ended up moving it to `src/types.js`. In the process I found a few duplicate types.